practice/00-HTML-CSS-basics/Expenses.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>page</title>
+</head>
+<body>
+List of expenses
+<ol>
```

Comment:
ol means that the list contains some kind of order. If there isn't any order in this list, you should use ul.